### DISCUSSION TITLE
Save the model

### DISCUSSION THREAD

**Discussion by schlinge2000** [ourownstory/neural_prophet#313] [Q&A - get help using NeuralProphet]
Hi all,
first, thanks a lot for your awesome work!
I wonder, if  there is already a functionality to store the fitted model including the RNN to reuse later?
Bests, Christian

**Comment by wangzhongqing** [comment]
Excuse me, can you save the model file to local？I want this function too。

**Comment by ourownstory** [comment]
Hi @schlinge2000 and @schlinge2000 currently no, but we would appreciate if you could help us implement this! See #27
